lib/services/platform-service.ts

```diff
@@ -310,6 +315,12 @@ export class PlatformService implements IPlatformService {
 			}
 		}).future<void>()();
 	}
+	
+	private isPlatformInstalled(platform: string): IFuture<boolean> {
+		return (() => {
+			return this.$fs.exists(path.join(this.$projectData.platformsDir, platform)).wait();
```

Comment:
you can return the future from fs.exists directly without wrapping it in new future